packages/smooth_app/lib/database/tmp.dart

```diff
@@ -0,0 +1,112 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:http/http.dart';
+import 'package:openfoodfacts/model/KnowledgePanels.dart';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/utils/QueryType.dart';
+import 'package:openfoodfacts/utils/UriHelper.dart';
+
+/// Fixes to off-dart.
+// TODO(monsieurtanuki): to be moved to off-dart
+class OpenFoodAPIClientTmp {
+  OpenFoodAPIClientTmp._();
+
+  /// Returns the product for the given barcode.
+  /// The ProductResult does not contain a product, if the product is not available.
+  /// ingredients, images and product name will be prepared for the given language.
+  ///
+  /// Please read the language mechanics explanation if you intend to show
+  /// or update data in specific language: https://github.com/openfoodfacts/openfoodfacts-dart/blob/master/DOCUMENTATION.md#about-languages-mechanics
+  static Future<ProductResult> getProduct(
```

Comment:
I don't think this is used

Reply:
You're right, `getProduct` from `tmp.dart` is not used, but that's the way it needs to be written when using new method `getProductString` - when the code is moved to off-dart.
Same thing for `getKnowledgePanels` / `getKnowledgePanelsString`

Reply:
that's brilliant, thank you